apps/dashboard/app/(app)/ratelimits/[namespaceId]/logs/filter.tsx

```diff
@@ -109,11 +109,14 @@ export const Filters: React.FC = () => {
             className="flex items-center h-8 gap-2 bg-background-subtle"
             onClick={() => {
```

Comment:
Let's wrap the whole content in startTransition instead of using an empty one. Also, the 'Clear Filters' function needs to handle 'after' and 'before' dates - they currently aren't being reset when clearing filters.

Reply:
ok shall i do the change ?

Reply:
Yeah, go ahead.